lib/sites/amazon.js

```diff
@@ -31,7 +31,7 @@ class AmazonSite {
       // a decimal point in the price... so we're just grabbing the dollar amount (not cents)
       // https://github.com/dylants/price-finder/issues/97
       // used in (at least) video games
-      '#price #newPrice .buyingPrice',
+      '#price #newPitchPrice .price-large',
```

Comment:
Based on the comment in the pull request, I'd suggest adding this new selector to the list, instead of modifying it, just in case the old selector works for other people.

```suggestion
      '#price #newPrice .buyingPrice',
      '#price #newPitchPrice .price-large',
```